pkg/async/schedule/flytescheduler/go_gf.go

```diff
@@ -0,0 +1,137 @@
+package flytescheduler
+
+import (
+	"context"
+	"fmt"
+	"github.com/flyteorg/flyteadmin/pkg/async/schedule/aws"
+	"github.com/flyteorg/flyteadmin/pkg/repositories/models"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/core"
+	"github.com/flyteorg/flytestdlib/logger"
+	"github.com/gogf/gf/os/gcron"
+	"github.com/gogf/gf/os/gtimer"
+	"github.com/robfig/cron"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type GoGF struct {
+}
+
+func (g GoGF) GetScheduleName(s models.SchedulableEntity) string {
+	return strconv.FormatUint(aws.HashIdentifier(core.Identifier{
```

Comment:
Ok as per the previous comment here https://github.com/flyteorg/flyteadmin/pull/228#discussion_r682547294
i would prefer to use a new function for generation hash with version to avoid backward compat issues with aws scheduler. Let me know if you agree

Reply:
My preference is to move the entire scheduler (the actual daemon) to be under
flyteadmin/scheduler/....

the way of doing flyteadmin/pkg/... is not encouraged by go conventions

Can we do that? and then refer to it here